apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -58,6 +58,16 @@ export default class TutorialDetail extends React.Component {
     // Disable body scrolling.
     $('body').css('overflow', 'hidden');
 
+    const languageComponent = (
+      <div style={styles.tutorialDetailLanguages}>
+        {this.props.item.language
+          .sort((a, b) => (a > b ? 1 : -1))
+          .map(language => (
+            <div>{language}</div>
```

Comment:
Minor nitpick, but since this is a list of items, it would be nice to use `<li></li>` since those are meant for displaying lists. Using HTML elements for their specific purpose helps improve our integration with a11y tools.

Reply:
Ah yes thanks. Updated to use an unordered list instead.

Reply:
> Ah yes thanks. Updated to use an unordered list instead.

We decided to not go this route actually and stick to a string. The list was considered harder to parse. [Related slack thread](https://codedotorg.slack.com/archives/C0SUN2SSF/p1635547981001300)